### ISSUE TITLE
Automatic Credit-Card-Type Detection not working as advertised.

### ISSUE THREAD

**Issue by FACamargo** [woothemes/storefront#451]
Can anybody give me pointers to solve this problem: I want/need to have the credit-card badges/icons to show up in the checkout page, in the credit-card number field when the user types in the numbers of his/her cards. 

I am using Wordpress 4.5.3, WooCommerce v2.64, and StoreFront 2.07 with the payment gateway Cielo WooCommerce - Solução Webservice v4.0.11, but I cannot make it work as advertised.

The credit-card badges do not show up when the user types in the number ot the credit-card.
Suposedly, this is a "feature" of StoreFront (listed in the changelog.md entry 2016.01.05 - 1.6.0 here at GitHub).

Here is my "minimal-test-web-store": http://www.solcentral.com.br/loja2/ which doesn't work

Here is another "minimal-test-web-store": http://wc.claudiosanches.com/ which DOES work. This store was created by Mr. Claudio Sanches, the creator of Cielo WooCommerce - Solução Webservice v4.0.11.

Both store use the same plugins, but I cannot find out why my store does not show the credit-card badges.

Please, HELP!

**Comment by jameskoster** [comment]
Hey,

Please post in the support forums for help, github is for reporting bugs :)

I looked at your site and I'm not sure why it's not working. The inspector shows the CSS is being loaded correctly but I don't see why the image isn't being loaded.

Like I said, please post in support and we can help further from there.

**Comment by FACamargo** [comment]
Dear James,

 

Thanks to your reply, it prompted me to look into my IIS server, and I managed to SOLVE the problem.  On my Windows 2008 R2 + IIS v7.5 Server, it had the mime-type for '.svg' files defined as "image/svg". I changed it to "image/svg+xml" and the problem was solved. Your comments below (in red) did the trick. Thank you very much for spending the time to look into my webstore pages. 

 

Sincerely,

 

Francisco A. Camargo

 <mailto:francisco.a.camargo@gmail.com> francisco.a.camargo@gmail.com

 

Cel: (49) 9173-2302 – Vivo

Tel: (49) 3241-4500 / 3241-6445
